Diplomas/DiplomaTest.php

```
@@ -0,0 +1,271 @@
+    }
+
+    public function testGetTeamQualResultsSeparatesDifferentTeams(): void
+    {
+        \FakeDb::on('/FROM Teams/', [
+            ['TeCoId' => 1, 'TeSubTeam' => 0, 'TeEvent' => 'RM', 'TeRank' => 1, 'TeScore' => 1900,
+                'EvEventName' => 'Recurve Men Team', 'EvMixedTeam' => 0,
+                'EnFullName' => 'Kowalski Jan', 'CoName' => 'Orzeł Warszawa', 'QuScore' => 650, 'TcOrder' => 1],
+            ['TeCoId' => 2, 'TeSubTeam' => 0, 'TeEvent' => 'RM', 'TeRank' => 2, 'TeScore' => 1850,
+                'EvEventName' => 'Recurve Men Team', 'EvMixedTeam' => 0,
+                'EnFullName' => 'Zielinski Adam', 'CoName' => 'Sokół Kraków', 'QuScore' => 620, 'TcOrder' => 1],
+        ]);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\FakeDb' in method 'testGetTeamQualResultsSeparatesDifferentTeams'. (undefined)

(StaticAccess)